app/models/validations/financial_validations.rb

```diff
@@ -121,13 +122,29 @@ def validate_charges(record)
     %i[scharge pscharge supcharg].each do |charge|
       maximum = CHARGE_MAXIMUMS.dig(charge, LANDLORD_VALUES[record.landlord], NEEDSTYPE_VALUES[record.needstype])
 
-      if maximum.present? && !weekly_value_in_range(record, charge, maximum)
+      if maximum.present? && !weekly_value_in_range(record, charge, 0, maximum)
         record.errors.add charge, I18n.t("validations.financial.rent.#{charge}.#{LANDLORD_VALUES[record.landlord]}.#{NEEDSTYPE_VALUES[record.needstype]}")
       end
     end
   end
 
-  def weekly_value_in_range(record, field, max)
-    record[field].present? && record.weekly_value(record[field]).present? && record.weekly_value(record[field]).between?(0, max)
+  def weekly_value_in_range(record, field, min, max)
+    record[field].present? && record.weekly_value(record[field]).present? && record.weekly_value(record[field]).between?(min, max)
+  end
+
+  def validate_rent_range(record)
+    return if record.startdate.blank?
+
+    window_end_date = Time.zone.local(record.year, 4, 1)
```

Comment:
I wonder if we should make this a `case_log` method? Think we do it in more than one place now. (i.e. `case_log.window_end_date`)

Reply:
There is a `form_name` method that gets the full form name, but happy to extract a smaller method out of there!